src/attestedclaim/AttestedClaim.spec.ts

```diff
@@ -72,6 +75,14 @@ describe('RequestForAttestation', () => {
     []
   )
 
+  it('compresses and decompresses the attested claims object', () => {
+    const compressedLegitimation = legitimation.compress()
```

Comment:
Perhaps `legitimation` could be renamed in attested claim? This confused me a bit. 😅

Reply:
Ahh, well this is because they are more than one attested claim. But, perhaps it could be changed.